### DISCUSSION TITLE
Struktura zespołu tłumaczy

### DISCUSSION THREAD

**Discussion by zwiastunsw** [irdpl/wcag#1] [Organizacja pracy]
Oczywista oczywistość: skuteczna organizacja pracy wymaga określenia ról w zespole i podziału zadań.
Jakie role są nam potrzebne? 
Jakie zadania przypisać poszczególnym rolom?
W jakiej roli/w jakich rolach widzisz siebie?

----
**Kilka propozycji**
redaktor techniczny/administrator
redaktor wydania
redaktor naukowy
redaktor - opiekun KS (kryterium sukcesu)
redaktor - opiekun Słownika / terminu

**Comment by PiotrtWitek** [comment]
To może ja zgłoszę się do roli: 

redaktor - opiekun KS (kryterium sukcesu)

**Comment by PiotrtWitek** [comment]
Uzupełniając, mógłbym się zająć: 

Consistent Help
i / lub
Redundant Entry

**Comment by zwiastunsw** [comment]
Dzięki za zgłoszenie.
Na razie mamy:
@BartekStepien: kryteria dotyczące fokusu (+powiązane terminy słownikowe)
@PiotrtWitek: Zbędne wpisy (+ powiązane terminy słownikowe) lub Spójna pomoc (+ powiązane terminy słownikowe)
@zwiastunsw: Spójna pomoc (+ powiązane terminy słownikowe)

**Reply by rotnicki** [reply]
W #26 zadeklarowałem zaopiekowanie się KS 2.4.11 Wygląd fokusu.

Chyba nam się nieopatrznie dwa wątki w tym samym obszarze tematycznym zrobiły.

**Comment by zwiastunsw** [comment]
W tym wątku wypowiadamy się tylko w kwestii organizacji zespołu. Deklaracjom służy #26

**Comment by EMF-MB** [comment]
Ja jestem dwujęzycznym prawnikiem, chętnie pomogę przy terminach prawniczych, skonfrontuję tłumaczenie z przepisami itd.
